staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -74,6 +74,7 @@ var (
 		// should be all lowercase and separated by underscores.
 		[]string{"verb", "dry_run", "group", "version", "resource", "subresource", "scope", "component", "client", "contentType", "code"},
 	)
+	// TODO(logicalhan): remove this deprecated metrics after fixing the dependencies
```

Comment:
@logicalhan did this deprecated metrics still need to be kept ?

Reply:
No need to kept.  What does `fixing the dependencies` mean? dependency trouble?

Reply:
ref https://github.com/kubernetes/kubernetes/pull/76496#issuecomment-496565529

Reply:
> would you mind adding back apiserver_request_count as well? We should be able to remove that one by 1.16, but 1.15 may be difficult for us.

We are in 1.17 now. So it's safe to remove them.

Reply:
cleaned